CODEOWNERS

```diff
@@ -102,7 +102,6 @@
 /src/lib/rc_pool/ @nholland94 @cmr
 /src/lib/receipt_chain_database_lib/ @johnwu93
 /src/lib/rocksdb/ @psteckler @nholland94
-/src/lib/root_history/ @cmr
```

Comment:
Can you add `/src/lib/transition_chain_witness` to this file and add yourself as a reviewer?